.vscode/launch.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
imo this don't have to be included

Reply:
I included that by mistake as I was debugging using VSCode. :)